scripts/dumpapp

```diff
@@ -53,19 +40,26 @@ def main():
   except KeyboardInterrupt:
     sys.exit(1)
 
-def read_frames(conn):
+def read_frames(sock):
   while True:
     # All frames have a single character code followed by a big-endian int
-    code = read_input(conn, 1, 'code')
-    n = struct.unpack('!L', read_input(conn, 4, 'int4'))[0]
+    code = read_input(sock, 1, 'code')
+    n = struct.unpack('!L', read_input(sock, 4, 'int4'))[0]
 
     if code == b'1':
       if n > 0:
-        sys.stdout.buffer.write(read_input(conn, n, 'stdout blob'))
+        sys.stdout.buffer.write(read_input(sock, n, 'stdout blob'))
```

Comment:
I'd prefer to tackle that in a separate diff, but yes I did still intend to have it work that way.